src/App.vue

```diff
@@ -187,13 +189,13 @@ export default {
   },
   watch: {
     isZapConfigSelected(val) {
-      if(val != true) {
-        this.$router.push({path: '/login'})
+      if (val != true) {
+        this.$router.push({ path: '/login' })
       } else {
         this.$router.push({ path: '/' })
         this.getAppData()
       }
-    }
-  }
+    },
```

Comment:
Not sure why the additional , here comes from.

Reply:
Auto-formatter. I'll let it do its thing.